.eslintignore

```diff
@@ -0,0 +1 @@
+lib
```

Comment:
Is it possible to lint `src/` directly [now that `test/` is gone](https://github.com/acdlite/redux-actions/commit/64b6a51bdc024d4d0af1dd67017a274881d785ef)?